2DALL.tscn

```diff
@@ -46,12 +46,12 @@
 animations = [ {
 "frames": [ ExtResource( 5 ) ],
 "loop": true,
-"name": "0",
+"name": @"0",
```

Comment:
You need to revert this line, because Godot has a bug and added @ before some strings which prevents editor from loading projects.